vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow/src/main/java/com/vaadin/flow/component/confirmdialog/ConfirmDialog.java

```diff
@@ -29,7 +28,9 @@
 import com.vaadin.flow.component.UI;
 import com.vaadin.flow.component.dependency.JsModule;
 import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.component.shared.InternalOverlayClassListProxy;
```

Comment:
Would it make sense to use the same package semantic flow is using? "shared.internal.ClassName" so that don't even see this class / interace when they navigate to the package? This could also allow to drop internal from the name

Reply:
Thanks, makes sense 👍  I will do that in a separate PR after this one lands.

Reply:
Created #4581 to move the class as you suggested.